core/src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -1710,6 +1715,8 @@ trait OptimisticTransactionImpl extends TransactionalWrite
     conflictChecker.checkConflicts()
   }
 
+  protected def getFirstAttemptVersion: Long = readVersion + 1L
```

Comment:
This is the first attempted _commit_ version?
(not sure if name change or just doc comment is better)

Reply:
The name was chosen to match `getNextAttemptVersion`. I will add a doc comment to make it clearer what it is.